src/main/java/org/junitpioneer/jupiter/RepeatFailedTest.java

```diff
@@ -62,7 +62,7 @@
 public @interface RepeatFailedTest {
 
 	/**
-	 * Specifies how often the test is executed at most.
+	 * Specifies how many times the test is executed at most.
```

Comment:
Please revert this change as it has nothing to do with the issue/PR (and will most probably result in merge conflicts of #236)